package.json

```diff
@@ -32,55 +32,56 @@
   ],
   "homepage": "https://www.jhipster.tech",
   "peerDependencies": {
-    "axios": "*",
-    "dayjs": "^1.10.4",
-    "react": "^18.0.0",
-    "react-dom": "^18.0.0",
-    "react-hook-form": "^7.30.0",
-    "reactstrap": "^9.0.3"
+    "axios": "0.27.2",
+    "dayjs": "1.11.3",
+    "react": "18.2.0",
+    "react-dom": "18.2.0",
+    "react-hook-form": "7.30.0",
+    "reactstrap": "9.1.3"
   },
   "peerDependenciesMeta": {
     "reactstrap": {
       "optional": true
     }
   },
   "dependencies": {
-    "lodash": "^4.17.21",
+    "lodash": "4.17.21",
```

Comment:
Fixed version at dependencies it will duplicate dependencies at the tree.
Makes the tree bigger and compiled application bigger due to duplicated code.